evm-tests/jsontests/tests/state.rs

```diff
@@ -54,9 +80,13 @@ const SKIPPED_CASES: &[&str] = &[
 	"stTransactionTest/HighGasPrice",
 	"stTransactionTest/HighGasPriceParis",
 	"stCreateTest/CreateTransactionHighNonce",
+	// KZG-precompile not supported
+	"stPreCompiledContracts/precompsEIP2929Cancun",
+	"stPreCompiledContracts/idPrecomps",
 ];
 
 fn should_skip(path: &Path) -> bool {
+	println!("{path:?}");
```

Comment:
This print seems like it doesn't provide much information. It will write all paths passed to this function to stdout, but without any context. Maybe it makes sense to at least include whether the path will be skipped or not.